packages/DrillDownTable/README.md

```diff
@@ -141,6 +141,26 @@ A boolean switch that makes the table render a custom
 
 A custom component that should be rendered when loading is true.
 
+#### getTdProps
+
+**optional**(`(cell: Cell) => Dictionary` | `undefined` = `undefined`)
+
+Use this to pass in a custom prop getter for the table cell elements. While the default for this is undefined, the table component
+does make use of a customTdProps getter that attaches a onClick handler that effects drilling down, This handler is only
+used when `useDrillDown = true` and `getTdProps` is undefined, otherwise if `getTdProps` is passed in then the component uses that as the click handler
```

Comment:
Should these be one continuous paragraph?  Seems like it is not

Reply:
I'll reformat